skawa_components/lib/list_wrapper/list_wrapper.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:async';
+import 'dart:html';
+
+import 'package:angular/core.dart';
+
+/// This component combined with the `SkawaForDirective` are used to be a minimal implementation for the
+/// (flyweight pattern)[https://en.wikipedia.org/wiki/Flyweight_pattern]. The concept is if you have a list with potentially
+/// 2-3000+ element with not small logic, it will a huge burden for the browser. Therefore this component will have some visible
+/// element (depends on height and the promised element height) and the other part will be filled up with 2 empty div
+/// (one on the top on e and the bottom). Pro it will much faster and have more fps then the normal `NgFor`,
+/// but you can't search in it with normal browser search (only the visible element are rendered).
+///
+///
+/// __Example usage:__
+///           <list-wrapper >
+///               <span *skawaFor="let item of list;itemHeight:75" style="height: 75px; display: block">{{item}}</span>
+///           </list-wrapper>
+///
+///
+/// __Events:__
+///
+/// - `reachBottom: Event` -- Published when the list will scrolled down to the bottom fully. I can be combined with
+///                            auto loading data from the backend.
+///
```

Comment:
```
/// - `reachBottom: Event` -- Published when the list is scrolled down to the bottom. 
///                           It can be used to trigger loading further elements from a service.
```

Reply:
changed